ui/README.md

```diff
@@ -1,2 +1,37 @@
-# Anbox Cloud - Cloud Gaming Demo
+# Web UI for cloud gaming demo
 
+The Web UI for clouding gaming demo is built with [flutter](https://flutter.dev/).
+
+## Requirements
+
+To setup Web development and build the UI, make sure that you have flutter installed.
+Please follow the [instructions](https://docs.flutter.dev/get-started/install/linux) to install the flutter on your machine.
+
+## Build
+
+To build the UI, simply run:
+
+    ```
+    flutter build web
+    ```
+
+This generates assets, js, html files that are required to run the Web UI and places all of them into the `build/web` folder.
+Please refer to the official [documentation](https://docs.flutter.dev/get-started/web) for more details on how to build Web apps with flutter.
+
+## Deploy
+
+Create a `static` folder under the backend service and copy everything that is in the `build/web` folder to the `static` folder.
```

Comment:
```suggestion
Create a `static` folder under the service directory and copy everything that is in the `build/web` directory to the `static` directory.

Reply:
Independently of where the instructions are, we should give the full folder path to avoid confusion. Where is the `build` folder created?

```suggestion
Create a `static` folder under the `backend/service` directory and copy everything that is in the `build/web` directory into it.
```